File: 10/ruby/Kiwi.rb
#Kiwi.rb | Defines the Kiwi class which inherits attributes and methods
#   from the Bird superclass.
#   #
#   # Begun by: Dr. Adams, for CS 214 at Calvin College.
#   # Completed by:
#   # Date:
#

require './Bird.rb'
require './WalkingBird.rb'

class Kiwi < WalkingBird

  def call
    "Hello Branden"
  end

end

File: 10/java/Kiwi.java
/* Penguin.java provides a Penguin subclass of Bird.
 *  *
 *   * Begun by: Prof. Adams, for CS 214 at Calvin College.
 *    * Completed by:
 *     * Date:
 *      ******************************************************/

public class Kiwi extends WalkingBird {

	/* explicit constructor
	 *          * Receive: name, a String 
	 *                   * PostCond: myName == name.
	 *                            */


	public Kiwi(String name)
	{
		super(name);
	}



	/* A Penguin Call
	 *          * Return: a Penguin call.
	 *                   */

	public String call()
	{
		return "Hello Branden.";
	}


}

